Rakefile

```diff
@@ -175,14 +231,33 @@ def create_packages(directory, platform, architecture, package_formats)
   Array(package_formats).map { |pkg| create_package(directory, pkg, platform, architecture) }
 end
 
-def create_package(tarball, pkg, platform, architecture)
-  output_name = [[PACKAGE_OUTPUT_NAME, architecture].join("_"), pkg].join(".")
-  extra_args  = EXTRA_ARGS[pkg] || ""
-  sh "fpm -s tar -t %s -f -n %s -v %s -a %s --license \"%s\" --vendor \"%s\" --maintainer \"%s\" --url \"%s\" --description \"%s\" --category \"%s\" --config-files %s -C %s -p %s %s %s" % [pkg, PACKAGE_NAME, VERSION, architecture, LICENSE, VENDOR, MAINTAINER, HOMEPAGE, DESCRIPTION, PACKAGE_CATEGORY, CONFIG_DEST, File.basename(tarball, ".tar.gz"), output_name, extra_args, tarball]
-  output_name
+def create_package(source, pkg, platform, architecture)
+  supported_by_fpm  = %w{deb rpm}
+  supported_by_nsis = %w{exe}
+  if supported_by_fpm.include?(pkg)
+    output_name = [[PACKAGE_OUTPUT_NAME, architecture].join("_"), pkg].join(".")
+    extra_args  = EXTRA_ARGS[pkg] || ""
+    sh "fpm -s tar -t %s -f -n %s -v %s -a %s --license \"%s\" --vendor \"%s\" --maintainer \"%s\" --url \"%s\" --description \"%s\" --category \"%s\" --config-files %s -C %s -p %s %s %s" % [pkg, PACKAGE_NAME, VERSION, architecture, LICENSE, VENDOR, MAINTAINER, HOMEPAGE, DESCRIPTION, PACKAGE_CATEGORY, CONFIG_DEST, File.basename(source, ".tar.gz"), output_name, extra_args, source]
+    output_name
+  elsif supported_by_nsis.include?(pkg)
+    nsis_script  = File.join("win32", "installer.nsis.erb")
+    vendor_files = File.join("win32", "vendor")
```

Comment:
Should part of the setup instructions be putting something into `win32/vendor` manually? There is nothing in there right now.